prbt_hardware_support/test/unittests/unittest_modbus_adapter_operation_mode.cpp

```diff
@@ -135,6 +163,50 @@ TEST_F(ModbusAdapterOperationModeTest, testModbusMsgOperationModeWrapperDtor)
   std::shared_ptr<ModbusMsgOperationModeWrapper> wrapper (new ModbusMsgOperationModeWrapper(msg_const_ptr, test_api_spec));
 }
 
+/**
+ * @brief Tests that an exception is thrown in case the Modbus message does not
+ * contain a register for the operation mode.
```

Comment:
Please fix test description. It does check for operation mode unknown.

Reply:
Done.